packages/frontend/core/src/components/explorer/context.ts

```
@@ -27,14 +27,7 @@ export const createDocExplorerContext = () =>
     selectMode$: new LiveData<boolean>(false),
     selectedDocIds$: new LiveData<string[]>([]),
     prevCheckAnchorId$: new LiveData<string | null>(null),
-    filters$: new LiveData<ExplorerPreference['filters']>([
-      {
-        type: 'system',
-        key: 'trash',
-        value: 'false',
-        method: 'is',
-      },
-    ]),
+    filters$: new LiveData<ExplorerPreference['filters']>([]),
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)